packages/fxa-settings/src/components/Ready/index.tsx

```diff
@@ -31,6 +31,7 @@ export type ReadyBaseIntegration = Pick<
   | 'getService'
   | 'getClientId'
   | 'isFirefoxClientServiceRelay'
+  | 'isFirefoxClientServiceNonSync'
```

Comment:
I am open to naming this something else. The `integration.isSync()` name feels simplistic in comparison... `isFirefoxNonSync`? `isNonSyncService`? Maybe we remove client and service to make it `isFirefoxNonSync`? I think we need the word "Firefox" or "browser" in there as just `isNonSync()` might get confusing since I might expect that to be true for a web or oauth redirect.

Reply:
Some more options:
`isFirefoxBrowserNonSyncService`
`isFirefoxServiceWithoutKeys`
😅

Reply:
`isFirefoxClientNonSyncService`